deps.ffmpeg/patches/FFmpeg/0003-FFmpeg-4.4.1-libaomenc.patch

```diff
@@ -16,4 +16,3 @@ index e739cd3b66..b3d458a686 100644
          return AVERROR(EINVAL);
 -- 
 2.25.1
-
```

Comment:
I think so? Patch is notoriously sensitive about trailing newlines.

Reply:
Apparently, that's a git format-patch signature. It could technically be omitted, but let's keep the patch changes minimal. If this whitespace change was needed, then fine.